request-server.go

```diff
@@ -150,6 +150,9 @@ func (rs *RequestServer) Serve() error {
 		delete(rs.openRequests, handle)
 		req.close()
 	}
+	if rs.pktMgr.alloc != nil {
```

Comment:
This might be more reliable as a `defer`? Just in case someone does an early `return`.